src/js/me-i18n.js

```diff
@@ -107,13 +105,22 @@
      * @return
      *   The translated string, escaped via i18n.methods.checkPlain()
      */
-    i18n.methods.t = function (str, options) {
+    i18n.methods.t = function (uid, options) {
+        var str;
 
         // Fetch the localized version of the string.
-        if (i18n.locale.strings && i18n.locale.strings[options.context] && i18n.locale.strings[options.context][str]) {
-            str = i18n.locale.strings[options.context][str];
+        if (i18n.locale.strings && i18n.locale.strings[options.context]) {
```

Comment:
Because it would be a redudant check with the following line which checks str
https://github.com/jalios/mediaelement/blob/2c5f19d56bc494f895ab1af97b9317b01d7d9536/src/js/me-i18n.js#L117